scripts/src/java/org/oppia/android/scripts/maven/GenerateMavenDependenciesList.kt

```diff
@@ -91,15 +90,63 @@ class GenerateMavenDependenciesList(
     if (licensesToBeFixed.isNotEmpty()) {
-      println("\nPlease verify the license link(s) for the following license(s) manually:")
+      val licenseToDependencyMap = findFirstDependencyWithBrokenLicense(
+        finalDependenciesList,
+        licensesToBeFixed
+      )
+      println(
+        """
+        Some licenses do not have their 'original_link' verified. To verify a license link, click
+        on the original link of the license and check if the link points to any valid license or
+        not. If the link does not point to a valid license (e.g - https://fabric.io/terms), set 
+        the 'is_original_link_invalid' field of the license to 'true'.
+         
+        e.g - 
+        license {
+          license_name: "Terms of Service for Firebase Services"
+          original_link: "https://fabric.io/terms"
+          is_original_link_invalid: true
+        }
+  
+        If the link does point to a valid license then choose the most appropriate category for 
+        the link:
+        1. scrapable_link: If the license text is plain text and the URL mentioned can be scraped
+                           directly from the original_link of the license. 
+                           e.g - https://www.apache.org/licenses/LICENSE-2.0.txt
+        2. extracted_copy_link: If the license text is plain text but it can not be scraped 
+                                directly from the original_link of the license.
+                                e.g - https://www.opensource.org/licenses/bsd-license
+        3. direct_link_only: If the license text is not plain text, it's best to display only the
+                             link of the license.
+                             e.g - https://developer.android.com/studio/terms.html
+        
+        After identifying the category of the license, modify the license to include one of the
+        above mentioned 'url'. 
+        
+        e.g - 
+        license {
+          license_name: "The Apache Software License, Version 2.0"
+          original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+          scrapable_link {
+            url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+          }
+        }
+        
+        Please verify the license link(s) for the following license(s) manually in 
+        maven_dependencies.textproto, note that only first dependency that contains the license 
+        needs to be updated and also re-run the script to update the license details at all places:
+        """.trimIndent()
+      )
       licensesToBeFixed.forEach {
         println("\nlicense_name: ${it.licenseName}")
         println("original_link: ${it.originalLink}")
         println("verified_link_case: ${it.verifiedLinkCase}")
         println("is_original_link_invalid: ${it.isOriginalLinkInvalid}")
+        println(
+          "First dependency that should be updated with the license: " +
+            "${licenseToDependencyMap[it]}\n"
```

Comment:
Just to clarify: after updating the first license & re-running the script, all the other occurrences will be automatically fixed, correct?

Reply:
Yes, right.